lib/types/spec.ex

```diff
@@ -6,5 +6,8 @@ defmodule Kadena.Types.Spec do
   @type reasons :: Keyword.t()
   @type error :: {:error, reasons()}
 
+  @callback new() :: struct()
   @callback new(any()) :: struct() | error()
+
+  @optional_callbacks new: 0
```

Comment:
@EdwinGuayacan @Odraxs Following the approach mentioned above, this callback is not necessary.